src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -382,9 +382,9 @@ private async Task<IReadOnlyList<RestoreSummary>> PerformNuGetV2RestoreAsync(Pac
                 cacheContext.NoCache = NoCache;
                 cacheContext.DirectDownload = DirectDownload;
 
-                PackageNamespacesConfiguration packageNamespacesConfiguration = PackageNamespacesConfiguration.GetPackageNamespacesConfiguration(Settings);
+                PackageSourceMapping packageSourceMappingConfiguration = PackageSourceMapping.GetPackageSourceMappingConfiguration(Settings);
```

Comment:
nit: var name. Not critical.

Reply:
This applies in many spots. You can consider changing these.

Reply:
fixed it in all the places using find and replace.

Reply:
Seems like there's still a bunch of these instances. Can you please double check that you committed those changes?

Reply:
Bumping this.

Reply:
I think I got it right this time and fixed in other places too.